packages/core/src/components/entity-title/_entity-title.scss

```diff
@@ -33,7 +32,6 @@
     flex-direction: row;
     gap: 0.5 * $pt-grid-size;
   }
-
```

Comment:
LGTM aside from this deleted line which is failing stylelint

Reply:
fixed!